main.py

```diff
@@ -36,14 +38,17 @@ def init():
 
     # Load config
     for path in config_paths:
-        try:
-            logging.debug('Loading config from {}'.format(path))
-            config = read_config(path)
-        except:
-            logging.exception('Failed to load config from {}'.format(path))
-        else:
-            logging.info('Successfully loaded config from {}'.format(path))
-            break
+        #Only try to load the config file if it's present
+        #(unclutters the logs)
+        if os.path.exists(path):
+            try:
+                logging.debug('Loading config from {}'.format(path))
+                config = read_config(path)
+            except:
+                logging.exception('Failed to load config from {}'.format(path))
```

Comment:
`logging.error` should be preferred here. exception is for exception: it prints the stacktrace which is irrelevant here

Reply:
Actually, it's relevant - the stacktrace contains, among list of stack frames (which are mostly irrelevant, in a way that most frames in a stacktrace are irrelevant except the one you're looking for ;-) ), the exception source and message. With non-existent config file filtering added, the exception message will either contain file access exception info (relevant) or JSON parsing exception info (relevant). Or... Did I understand your statement correctly?

Reply:
You understood correctly, and you're right about it.